hedy.py

```diff
@@ -2823,7 +2823,9 @@ def transpile(input_string, level, lang="en", skip_faulty=True):
         transpile_result = transpile_inner(input_string, level, lang, populate_source_map=True)
 
     except Exception as original_error:
-        if getenv('ENABLE_SKIP_FAULTY', False) and skip_faulty:
+        hedy_amount_lines = len(input_string.split('\n'))
+
+        if getenv('ENABLE_SKIP_FAULTY', False) and skip_faulty and hedy_amount_lines > 1:
```

Comment:
@Felienne is the `ENABLE_SKIP_FAULTY` env variable set to true on production? Because I had to set it myself to see the effect of the changes!

Reply:
The variable is set to false in production! This is because this feature changes a lot of the core of Hedy, so we want to ship it to teachers once we test extensively and also we want to prepare a proper announcement.

Reply:
Hey Jesus, thanks for replying and that makes sense.